core/src/main/java/apoc/trigger/Trigger.java

```diff
@@ -48,38 +52,63 @@ public TriggerInfo( String name, String query, Map<String,Object> selector, Map<
     @Context public GraphDatabaseService db;
 
     @Context public TriggerHandler triggerHandler;
+    
+    @Context public Log log;
```

Comment:
Secondo me puoi fare fare anche:
```suggestion
    @Context public Log log;
    
    @Context public DatabaseManagementService dbms;
```

Ed eliminare `ApocConfig#getDatabase`

Reply:
Purtroppo non va, dice `....DatabaseManagementService is not a known injectable component`